app/javascript/gobierto_visualizations/webapp/containers/contracts/ContractsShow.vue

```diff
@@ -91,13 +153,46 @@ export default {
       process_type: '',
       permalink: '',
       assignee_routing_id: '',
-      labelAsignee: I18n.t('gobierto_visualizations.visualizations.contracts.assignee'),
-      labelTenderAmount: I18n.t('gobierto_visualizations.visualizations.contracts.tender_amount'),
-      labelContractAmount: I18n.t('gobierto_visualizations.visualizations.contracts.contract_amount'),
-      labelStatus: I18n.t('gobierto_visualizations.visualizations.contracts.status'),
-      labelProcessType: I18n.t('gobierto_visualizations.visualizations.contracts.process_type'),
-      labelContractType: I18n.t('gobierto_visualizations.visualizations.contracts.contract_type'),
-      labelPermalink: I18n.t('gobierto_visualizations.visualizations.contracts.permalink')
+      contractor: '',
+      contract_type: '',
+      start_date: '',
+      end_date: '',
+      award_date: '',
+      batch_number: '',
+      minor_contract: '',
+      open_proposals_date: '',
+      submission_date: '',
+      cpvs: '',
+      category_title: '',
+      labelAwardingEntity: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.awarding_entity') || '',
+      labelAssigneeDescription: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.assignee_description') || '',
+      labelContractAmount: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.contract_amount') || '',
+      labelType: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.type') || '',
+      labelProcess: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.process') || '',
+      labelTender: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.tender') || '',
+      labelAwarding: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.awarding') || '',
+      labelBidDescription: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.bid_description') || '',
+      labelBiddersDescription: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.bidders_description') || '',
+      labelEntity: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.entity') || '',
+      labelQuestionDescription: I18n.t('gobierto_visualizations.visualizations.contracts.contracts_show.question_description') || '',
+      labelStatus: I18n.t('gobierto_visualizations.visualizations.contracts.status') || '',
+      labelCategory: I18n.t('gobierto_visualizations.visualizations.subsidies.category') || '',
+      filterContractsBatchs: []
+    }
+  },
+  computed: {
+    hasBatch() {
+      return this.batch_number > 0
+    },
+    minorContract() {
+      return this.minor_contract === 'f'
+    },
+    showArrowDate() {
+      return this.submission_date && this.open_proposals_date
+    },
+    numberOfProposals() {
+      const tenderContract = this.tendersData.filter(({ cpvs }) => cpvs === this.cpvs)
+      return tenderContract[0]?.number_of_proposals
```

Comment:
When you use the first item of an array, you must explain why so

Reply:
Weel, in this case, we only get one object, I've changed it by destructuration